exercises/two-fer/two-fer.spec.js

```diff
@@ -0,0 +1,18 @@
+
+var TwoFer = require('./two-fer');
+
+describe('Two Fer', function () {
+  var twoFer = new TwoFer();
+
+  it('gives one to you if no parameter given', function () {
+    expect(twoFer.twoFer()).toEqual('One for you, one for me.');
+  });
+
+  it('gives one to Alice if \'Alice\' is given', function () {
```

Comment:
We would like to see all tests but the first skipped, which you can do by changing `it` to `xit` in lines 11 and 15.